File: 프로그래머스/1/82612. 부족한 금액 계산하기/부족한 금액 계산하기.cs
using System;

class Solution
{
    public long solution(int price, int money, int count)
    {
        long p = price;
        long m = money;
        for(int i=1; i<=count; i++)
        {
            m -= p * i;
        }
        return m > 0 ? 0 : m * (-1);
    }
}

File: 프로그래머스/1/82612. 부족한 금액 계산하기/부족한 금액 계산하기.cpp
using namespace std;

long long solution(int price, int money, int count)
{
    long long ans = 0;
    long long sum = 0;
    for(int i=1; i<=count; i++)
    {
        sum += price * i;
    }
    if(sum > money) { ans = sum - money; }
    else { ans = 0; }
    return ans;
}